ENGAGEHF/Bluetooth/Views/MeasurementRecordedView.swift

```diff
@@ -0,0 +1,123 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct MeasurementHeader: View {
+    @Environment(\.dismiss) var dismiss
+    @Binding var viewState: ViewState
+    
+    
+    var body: some View {
+        ZStack {
+            HStack {
+                Button(action: {
+                    dismiss()
+                }) {
+                    Text("DISMISS_BUTTON")
+                        .foregroundStyle(Color.accentColor)
+                }
+                    .buttonStyle(PlainButtonStyle())
+                    .padding(.trailing, 10)
+                    .disabled(viewState != .idle)
+                    .accessibilityLabel("Dismiss")
+                
+                Spacer()
+            }
+            
+            HStack {
+                Spacer()
+                
+                Text("Measurement Recorded")
+                    .font(.title2)
+                
+                Spacer()
+            }
```

Comment:
Looking at the UI, the "custom Navigation Bar" might not be the best way we could display this. It might make sense to make the text a bit smaller and put it right above (or maybe even better below) the larger text displaying the recorded weight?